fhir-server/src/test/java/com/ibm/fhir/server/test/FHIRRestHelperTest.java

```diff
@@ -0,0 +1,987 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.server.test;
```

Comment:
It'd be great to have an integration test in fhir-server-test.

Reply:
Are the tests in com.ibm.fhir.server.test.Bundletest not sufficient?

Reply:
sounds fair Lee